h/static/extension/config.js

```diff
@@ -1,8 +0,0 @@
-function hypothesisConfig() {
-  // Pages on our site can include a meta tag to trigger specific behaviour
-  // when the extension loads.
-  var hypothesisIntent = document.querySelector('[name="hypothesis-intent"]');
```

Comment:
Also, if it's gone from here it should be gone from `h/templates/help.html.jinja2`